docs/modules/SDDs/pages/getting-started.adoc

```diff
@@ -0,0 +1,269 @@
+= Getting Started with Project Syn
+
+This guide helps to get started with the various Project Syn tools.
+
+== Requirements
+
+* A local Kubernetes cluster (k3s), managed with https://k3d.io/[k3d]
+* A https://gitlab.com[GitLab.com] account with an https://gitlab.com/profile/keys[SSH key configured]
+* `curl`
+
+== Kickstart Lieutenant
+
+Create a namespace to host the Operator and the API
+[source,shell]
+----
+kubectl create namespace lieutenant
+----
+
+=== Install Lieutenant Operator
+
+Install *Lieutenant Operator* with the following commands:
+
+[source,shell]
+----
+# CRDs
+kubectl -n lieutenant apply -f https://raw.githubusercontent.com/projectsyn/lieutenant-operator/master/deploy/crds/syn.tools_clusters_crd.yaml
```

Comment:
With https://github.com/projectsyn/lieutenant-api/pull/61 and https://github.com/projectsyn/lieutenant-operator/pull/71 merged you can now use
```
kubectl apply -k github.com/projectsyn/lieutenant-operator/deploy
kubectl apply -k github.com/projectsyn/lieutenant-operator/deploy/crds
```

and

```
kubectl apply -k github.com/projectsyn/lieutenant-api/deploy
```